app-text/zathura/zathura-0.5.3.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson virtualx xdg
+
+DESCRIPTION="A highly customizable and functional document viewer"
+HOMEPAGE="https://pwmt.org/projects/zathura/"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.pwmt.org/pwmt/${PN}.git"
+	EGIT_BRANCH="develop"
+else
+	SRC_URI="
+		https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz
+		https://cdn.turret.cyou/e28b2f940d1a19a74ecbfd80ea4477c5ea9ac627/${P}-manpages.tar.xz
+	"
+	KEYWORDS="~amd64 ~arm ~riscv ~x86 ~amd64-linux ~x86-linux"
+fi
+
+LICENSE="ZLIB"
+SLOT="0/5.6"
+IUSE="seccomp sqlite synctex test"
+
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=dev-libs/girara-0.4.1
+	>=dev-libs/glib-2.50:2
+	dev-libs/json-glib
+	sys-apps/file
+	>=sys-devel/gettext-0.19.8
+	x11-libs/cairo
+	>=x11-libs/gtk+-3.22:3[X]
```

Comment:
What I meant was 
```
DEPEND="
	>=x11-libs/gtk+-3.22:3
	...
	test? ( x11-libs/gtk+[X] )
"
```

Reply:
ic. i'll do what you've done over at girara as that does make the most sense at the moment and coincides with the devmanual.